txpipe/twopoint.py

```diff
@@ -1063,17 +1324,20 @@ def write_output(self, data, meta, results):
     def write_output_plot(self, results):
         import matplotlib.pyplot as plt
         d = results[0]
+        dvalue = d.object.xi
+        derror = np.sqrt(d.object.varxi)
+        dtheta = np.exp(d.object.meanlogr)
 
         fig = self.open_output('gammat_bright_stars_plot', wrapper=True)
 
         # compute the mean and the chi^2/dof
         flat1 = 0
```

Comment:
Why are we subtracting off zero? Looks like this isn't from your PR, but a good place to fix it I think.

Reply:
No clue, I tried to clean it up